src/js/mockedData.jsx

```diff
@@ -389,6 +389,144 @@ const SUBSTITUTIONS_MOCKS = {
 
 const MOVEMENT_NUMBER = 'ABC123';
 
+const BIN_LOCATION_MOCKS = [
+  { value: 'Receiving', label: 'Receiving' },
+  { value: 'Bin 1', label: 'Bin 1' },
+  { value: 'Bin 2', label: 'Bin 2' },
+];
+
+const RECEIPT_MOCKS = {
```

Comment:
NOTE: This is actually going to be a shipment, not a receipt. Receipts will be what we save to when the user performs a partial receipt.

Reply:
This does not need to be changed, I just wanted to point it out for clarity.